Workflow/Modules/Script.py

```diff
@@ -68,7 +68,8 @@ def _setCommand( self ):
     if os.path.exists( os.path.basename( self.executable ) ):
       self.executable = os.path.basename( self.executable )
       if not os.access( '%s/%s' % ( os.getcwd(), self.executable ), 5 ):
-        os.chmod( '%s/%s' % ( os.getcwd(), self.executable ), stat.S_IRWXU | stat.S_IRGRP | stat.S_IXGRP | stat.S_IROTH | stat.S_IXOTH )
+        os.chmod( '%s/%s' % ( os.getcwd(), self.executable ),
```

Comment:
Mind adding a comment with the octal form of all these flags ? :D

Reply:
Linking to https://docs.python.org/2/library/stat.html#stat.S_IRWXU is enough?

Reply:
well, sure, we can understand from it, but most human understand better ```640``` than the flag sequence